pkg/orchestrator/orchestrator.go

```diff
@@ -74,7 +74,7 @@ type PipelineService interface {
 	// Stop initiates a graceful shutdown of the given pipeline.
```

Comment:
we should update the docs for this method, it only talks about the graceful shutdown